deployments/cardano-explorer-env-production.nix

```diff
@@ -0,0 +1,12 @@
+with (import ./../lib.nix);
+
+let
+  nodeProdConf = import ./../modules/cardano-node-prod.nix;
+in {
+  sl-explorer = nodeProdConf 40 region;
```

Comment:
Is the significance of `40` documented somewhere? The earliest use I found is at https://github.com/input-output-hk/iohk-nixops/blob/c92d26a327cf620a967702438ac575106fa115aa/deployments/cardano-explorer.nix#L15

Is it an arbitrary index `> 13` assigned to the explorer host? (13 because we have 14 zero-indexed cardano node hosts.)